### PR TITLE
Fix to prevent deploys with keys used for blessed (standard) contracts

### PR DESCRIPTION
**Pull request by tgrospic**
## Overview
This is security fix already applied to main net validators as v0.9.25.2 release.

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Review by nzpr — APPROVED** [review]

File: casper/src/test/scala/coop/rchain/casper/genesis/contracts/StandardDeploysSpec.scala [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package coop.rchain.casper.genesis.contracts
+
+import coop.rchain.crypto.codec.Base16
+import org.scalatest.FlatSpec
+
+class StandardDeploysSpec extends FlatSpec {
```
Comment (zsluedem) [inline casper/src/test/scala/coop/rchain/casper/genesis/contracts/StandardDeploysSpec.scala:6]: What's the intention on this test?

**Review by zsluedem — APPROVED** [review]
LGTM

File: casper/src/test/scala/coop/rchain/casper/genesis/contracts/StandardDeploysSpec.scala (hunk 1 above)
Comment (tgrospic) [reply casper/src/test/scala/coop/rchain/casper/genesis/contracts/StandardDeploysSpec.scala:6]: Just to record in the test logs what are the corresponding public keys.

**Comment by tgrospic** [comment]
bors merge

**Pull request merged**